backoffice/backoffice-domain-service/ballerina/modules/backoffice/backoffice-api_service.bal

```diff
@@ -0,0 +1,117 @@
+import ballerina/http;
+import ballerina/io;
+import ballerina/lang.value;
+import backoffice_service.org.wso2.apk.apimgt.api as api;
+import backoffice_service.org.wso2.apk.apimgt.rest.api.backoffice.v1.common.impl as backoffice;
+
+configurable int BACKOFFICE_PORT = 9443;
+
+listener http:Listener ep0 = new (BACKOFFICE_PORT);
+
+@http:ServiceConfig {
+    cors: {
+        allowOrigins: ["*"],
+        allowCredentials: true,
+        allowHeaders: ["*"],
+        exposeHeaders: ["*"],
+        maxAge: 84900
+    }
+}
+
+service /api/am/backoffice/v1 on ep0 {
```

Comment:
we can remove v1

Reply:
addressed with f0cddb250859adb268acb465a0f427f1da0e5766